Meta/shell_include.sh

```diff
@@ -6,10 +6,19 @@
 # NOTE: If using another privilege escalation binary make sure it is configured or has the appropiate flag
 #       to keep the current environment variables in the launched process (in sudo's case this is achieved
 #       through the -E flag described in sudo(8).
-SUDO="sudo -E"
-
 if [ "$(uname -s)" = "SerenityOS" ]; then
     SUDO="pls -E"
+elif command -v sudo >/dev/null; then
+	SUDO="sudo -E"
+elif command -v doas >/dev/null; then
+	if [ "$SUDO_UID" = '' ]; then
+		export SUDO_UID=$(id -u) SUDO_GID=$(id -g)
+	fi
+	# To make doas work, you have to make sure you use the "keepenv" flag in doas.conf
+	SUDO="doas"
+else
+	echo "You need sudo, doas or pls to build Serenity..." >&2
+	exit 1
```

Comment:
I think we can use the alias below: `die "You need sudo, doas or pls to build Serenity..."`

Reply:
Will need to reorder the functions, but doable; I don't know why I didn't do that :p